### ISSUE TITLE
GeneralisedNewtypeDeriving not recognised by stylish-haskell

### ISSUE THREAD

**Issue by newhoggy** [jaspervdj/stylish-haskell#295]
The British spelling `GeneralisedNewtypeDeriving` is not recognised by `stylish-haskell`, so `stylish-haskell` fails to format the source file when only the British spelling is used.

```
Language.Haskell.Stylish.Parse.parseModule: could not parse ./cardano-config/src/Cardano/TracingOrphanInstances/Shelley.hs: ParseFailed (SrcLoc "<unknown>.hs" 159 18) "GeneralizedNewtypeDeriving language extension is not enabled. Please add {-# LANGUAGE GeneralizedNewtypeDeriving #-} pragma at the top of your module."
```

**Comment by newhoggy** [comment]
### Work around

Add `GeneralizedNewtypeDeriving` to `.stylish-haskell.yaml`

**Comment by jaspervdj** [comment]
This should be fixed by #307, please reopen if it’s still happening?
